src/vs/base/browser/ui/checkbox/checkbox.ts

```diff
@@ -174,3 +180,46 @@ export class Checkbox extends Widget {
 		this.domNode.setAttribute('aria-disabled', String(true));
 	}
 }
+
+export class SimpleCheckbox extends Widget {
```

Comment:
How about "ActualCheckbox" or "CheckboxWithCheckmark" 🤣

Reply:
Can't tell how much /s in this comment

Reply:
Lots